### DISCUSSION TITLE
index.html & exercise style

### DISCUSSION THREAD

**Discussion by PabloAsensio** [JdeRobot/RoboticsAcademy#1517] [Q&A] [answered]
Hello! 

I am trying to make changes to the web such as button color heat, background color and others, but no changes are being made. I have searched for the index.html but why are there 5? What is the one that loads the web at the beginning?

I've tried changing exercise.html to include messages in the web terminal and they don't show up. As I see that there is duplication of code, is the exercise.html that is executed in [1] or in [2]?

[1] - ./exercises/templates/exercises/follow_line/exercise.html
[2] - ./exercises/static/exercises/follow_line/web-template/exercises.html

*All the style changes were made to a .css file that was in the .html directory

![unnamed](https://user-images.githubusercontent.com/44872361/150568759-ac93d47a-2a05-4fa2-980c-92688479051c.png)

**Comment by shashwat623** [comment, marked as answer]
Hey @PabloAsensio it depends on how you are executing the exercise. The  [[1] - ./exercises/templates/exercises/follow_line/exercise.html ](https://github.com/JdeRobot/RoboticsAcademy/tree/master/exercises/templates/exercises/follow_line) is the template used when running the exercise via the docker container.

The [[2] - ./exercises/static/exercises/follow_line/web-template/exercises.html](https://github.com/JdeRobot/RoboticsAcademy/tree/master/exercises/static/exercises/follow_line/web-template) is the template used when running the exercise independently i.e without docker.

As far as making changes to the css or js files go, you need to do that from here  [_exercises/static/exercises/follow_line/web-template/assets/_](https://github.com/JdeRobot/RoboticsAcademy/tree/master/exercises/static/exercises/follow_line/web-template/assets). As of now this is the common css and js files source for both the docker and non docker template.
Hope that helps.

**Reply by PabloAsensio** [reply]
Seems that the path is to make changes to the css is [/static/common/css/](https://github.com/JdeRobot/RoboticsAcademy/tree/master/static/common/css)
